lib/record_store/provider/ns1.rb

```diff
@@ -0,0 +1,192 @@
+require 'ns1'
```

Comment:
We don't seem to be using the `ns1` gem in here, however we are using the NS1 client built in the other file, this should thus be changed into a `require_relative` of the client

Reply:
Great,
it will be updated like this!
```ruby
require_relative 'ns1'

module RecordStore
  class Provider::NS1 < Provider
  (snip)
```

Reply:
You mean `require_relative 'ns1/client'` ? Or does `require_relative 'ns1'` include automatically everything that's in the `ns1` directory ? (Ruby noob :) )

Reply:
I see what you mean! Will be updated as suggested (Ruby noob here :) )